### ISSUE TITLE
sortedUniqBy triggers no-double-unwrap rule

### ISSUE THREAD

**Issue by mbernardeau** [wix/eslint-plugin-lodash#143]
The rule `no-double-unwrap` is trigerred by the following code:

```
_(itemUnits)
          .compact()
          .sortBy('code')
          .sortedUniqBy('code')
          .value()
```

If I replace `sortedUniqBy` by `uniqBy`, rule is no longer triggered.

From lodash documentation:

> This method is like _.uniqBy except that it's designed and optimized for sorted arrays.

**Comment by ganimomer** [comment]
You're right! Fixing.

**Comment by ganimomer** [comment]
Fixed in `v2.3.7`. Thanks for reporting!

**Comment by mbernardeau** [comment]
@ganimomer 
Thanks a lot, that was quick !
